x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/registry/ModelRegistryImpl.java

```diff
@@ -55,49 +57,21 @@
 
 import static org.elasticsearch.core.Strings.format;
 
-public class ModelRegistry {
+public class ModelRegistryImpl implements ModelRegistry {
```

Comment:
I can't think of a better name than `ModelRegistryImpl`. Usually the name would reflect how the concrete class is implemented so this would be `IndexBackedModelRegistry` or similar but that name doesn't really aid our understanding.